File: GitNexus/gitnexus/test/fixtures/lang-resolution/java-overload-dispatch/SqlRepository.java
public class SqlRepository implements Repository {
    public String find(int id) {
        return "found-by-id";
    }

    public String find(String name, boolean exact) {
        return "found-by-name";
    }

    public void save(String data) {
        System.out.println(data);
    }
}

File: GitNexus/gitnexus/test/fixtures/lang-resolution/csharp-overload-dispatch/SqlRepository.cs
public class SqlRepository : IRepository {
    public string Find(int id) {
        return "found-by-id";
    }
    public string Find(string name, bool exact) {
        return "found-by-name";
    }
    public void Save(string data) {
        Console.WriteLine(data);
    }
}
